pom.xml

```diff
@@ -201,6 +214,15 @@
                     </container>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Do we need to specify this because it is not sufficiently well specified in the parent pom?